### PR TITLE
Re-enable pprof for mixer with args consistent with Pilot

### PR DESCRIPTION
**Pull request by mandarjog**
Enable pprof at http://localhost:9093/debug/pprof/* with command line args consistent with Pilot.

Signed-off-by: Mandar U Jog <mjog@google.com>

### CODE REVIEW

File: mixer/pkg/server/monitoring.go [hunk 1]
```diff
@@ -61,6 +62,14 @@ func startMonitor(port uint16) (*monitor, error) {
 		}
 	})
 
+	if enableProfiling {
```
Comment (geeknoid) [inline mixer/pkg/server/monitoring.go]: Can you create a unit test for this? The server package has 100% coverage, it'd be nice to stay there :-)

**Comment by yutongz** [comment]
/test e2e-simple
/test e2e-bookInfo

We renamed e2e-smoke to e2e-bookInfo and added e2e-simple
ref: https://github.com/istio/istio/pull/3529

**Comment by douglas-reid** [comment]
@mandarjog do we want some sort of integration test to validate this works as expected to go along with the unit test?

**Comment by ldemailly** [comment]
/lgtm
#3587 to see what's up with the linter error

**Comment by ldemailly** [comment]
sorry to put that on you mandar but can you fix the linter issues (#3471 is the cause of this problem, master is broken because the pr that added the lint errors was green because the linters got 'fixed' in between)
otherwise I'll get to it after I fix #3585

**Comment by ldemailly** [comment]
linter are fixed (#3591 ) so you should be good to go

/test istio-pilot-e2e

File: mixer/pkg/server/monitoring.go (hunk 1 above)
Comment (ldemailly) [reply mixer/pkg/server/monitoring.go]: the default being `true`, it's covered

**Pull request merged**